src/Qir/Controller/Program.cs

```diff
@@ -31,6 +40,15 @@ static void Main(string[] args)
             };
 
             rootCommand.AddOption(outputOption);
+
+            var libraryDirectoryOption = new Option<DirectoryInfo>(
+            aliases: new string[] { "--libraryDirectory" })
+            {
+                Description = "Path to the directory containing the libraries that must be linked to the driver executable.",
+                IsRequired = true
+            };
+
+            rootCommand.AddOption(libraryDirectoryOption);
```

Comment:
We will also need to add a `--includeDirectory` option which will be used by the executable generator in a similar way that the `--libraryDirectory` is used.